waspc/examples/todoApp/src/client/pages/auth/Login.js

```diff
@@ -2,7 +2,8 @@ import React from 'react'
 import { Link } from 'react-router-dom'
 
 import LoginForm from '@wasp/auth/forms/Login'
-// import { GoogleSignInButton } from '@wasp/auth/buttons/Google'
+import { SignInButton as GoogleSignInButton } from '@wasp/auth/buttons/Google'
+import { SignInButton as GitHubSignInButton } from '@wasp/auth/buttons/GitHub'
```

Comment:
Are we good with having the exports from each type of button be the same name, and they can just rename on import? It greatly simplifies the templates.